package.json

```diff
@@ -24,7 +24,7 @@
     "mocha": "^8.3.1",
     "prebuild": "^10.0.1",
     "superstring": "^2.4.2",
-    "tree-sitter-javascript": "https://github.com/tree-sitter/tree-sitter-javascript.git#master"
+    "tree-sitter-javascript": "https://github.com/tree-sitter/tree-sitter-javascript.git#936d976a782e75395d9b1c8c7c7bf4ba6fe0d86b"
```

Comment:
newer versions of tree-sitter-javascript fail with an "incompatible language version 14" error. In #127 I upgrade tree-sitter instead.